// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NoRegressionStack > not a function (with cause) 1`] = `
[Error: Property failed after 1 tests
{ seed: 42, path: "0:0", endOnFailure: true }
Counterexample: [0]
Shrunk 1 time(s)

Hint: Enable verbose mode in order to have the list of all failing values encountered during the run]
`;

exports[`NoRegressionStack > not a function 1`] = `
[Error: Property failed after 1 tests
{ seed: 42, path: "0:0", endOnFailure: true }
Counterexample: [0]
Shrunk 1 time(s)
Got TypeError: v is not a function
    at packages/fast-check/test/e2e/NoRegressionStack.spec.ts:?:?
    at Property.predicate (packages/fast-check/src/check/property/Property.ts:?:?)
    at Property.run (packages/fast-check/src/check/property/Property.generic.ts:?:?)
    at runIt (packages/fast-check/src/check/runner/Runner.ts:?:?)
    at check (packages/fast-check/src/check/runner/Runner.ts:?:?)
    at Module.assert (packages/fast-check/src/check/runner/Runner.ts:?:?)
    at packages/fast-check/test/e2e/__test-helpers__/StackSanitizer.ts:?:?

Hint: Enable verbose mode in order to have the list of all failing values encountered during the run]
`;

exports[`NoRegressionStack > return false (with cause) 1`] = `
[Error: Property failed after 2 tests
{ seed: 42, path: "1:0:1:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0", endOnFailure: true }
Counterexample: [0,1]
Shrunk 32 time(s)

Hint: Enable verbose mode in order to have the list of all failing values encountered during the run]
`;

exports[`NoRegressionStack > return false 1`] = `
[Error: Property failed after 2 tests
{ seed: 42, path: "1:0:1:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0", endOnFailure: true }
Counterexample: [0,1]
Shrunk 32 time(s)
Got error: Property failed by returning false
    at Property.run (packages/fast-check/src/check/property/Property.generic.ts:?:?)
    at runIt (packages/fast-check/src/check/runner/Runner.ts:?:?)
    at check (packages/fast-check/src/check/runner/Runner.ts:?:?)
    at Module.assert (packages/fast-check/src/check/runner/Runner.ts:?:?)
    at packages/fast-check/test/e2e/__test-helpers__/StackSanitizer.ts:?:?
    at Proxy.methodWrapper (node_modules/chai/chai.js:?:?)
    at packages/fast-check/test/e2e/NoRegressionStack.spec.ts:?:?

Hint: Enable verbose mode in order to have the list of all failing values encountered during the run]
`;

exports[`NoRegressionStack > throw (with cause) 1`] = `
[Error: Property failed after 2 tests
{ seed: 42, path: "1:0:1:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0", endOnFailure: true }
Counterexample: [0,1]
Shrunk 32 time(s)

Hint: Enable verbose mode in order to have the list of all failing values encountered during the run]
`;

exports[`NoRegressionStack > throw 1`] = `
[Error: Property failed after 2 tests
{ seed: 42, path: "1:0:1:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0", endOnFailure: true }
Counterexample: [0,1]
Shrunk 32 time(s)
Got error: a must be >= b
    at packages/fast-check/test/e2e/NoRegressionStack.spec.ts:?:?
    at Property.predicate (packages/fast-check/src/check/property/Property.ts:?:?)
    at Property.run (packages/fast-check/src/check/property/Property.generic.ts:?:?)
    at runIt (packages/fast-check/src/check/runner/Runner.ts:?:?)
    at check (packages/fast-check/src/check/runner/Runner.ts:?:?)
    at Module.assert (packages/fast-check/src/check/runner/Runner.ts:?:?)
    at packages/fast-check/test/e2e/__test-helpers__/StackSanitizer.ts:?:?

Hint: Enable verbose mode in order to have the list of all failing values encountered during the run]
`;
